src/components/form-create-account/form-create-account.js

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from "react";
```

Comment:
I don't see why we would have a component like this. This is not scalable. It's hard to say if more apps will use it. Even if they do now, if one needs changes, it will affect the rest. I would remove this one and only export the granular components `FormInput`, `Radio`, etc.

Reply:
This component was the request :-) - all other components were created to support this one.

This component was supposed to be used in a couple of applications - see [issue #30 
](https://github.com/code4romania/taskforce-fe-components/issues/30 )

Please have a chat with @aniri and let me know if this is needed or not.

Reply:
Hey @danut0nline and thanks for your help! @moonflare does have a point, it would make more sense to keep only the granular components in this project and move the form building logic in the applications. The form will most likely look the same in both apps now, but it might change in the future and it would be harder to update it if we keep it in the components project. But we could move all the form related logic to the https://github.com/code4romania/stam-acasa project. :smile:

Reply:
I see - I'll remove the form component